libraries/AP_Scripting/examples/6DoF_roll_pitch.lua

```diff
@@ -0,0 +1,54 @@
+-- This script sets the target roll and pitch to the current value when arming or when E-stop is removed
+local e_stop = rc:find_channel_for_option(31)
+local sw = rc:find_channel_for_option(300)
+local active = false
+
+function update() -- this is the loop which periodically runs
+
+  -- motor state
+  local current_active = false
```

Comment:
this is a script so we are not as strict as with the c++ code but maybe rename "current_active" to "motors_armed" or "motors_spinning"?  Sorry to be so annoying about names but it saves the reader time if we pick good names.